header.cmake

```diff
@@ -147,20 +147,20 @@ ENDMACRO(_SETUP_PROJECT_HEADER)
 FUNCTION(GENERATE_CONFIGURATION_HEADER
     HEADER_DIR FILENAME LIBRARY_NAME EXPORT_SYMBOL)
 
-  IF(${PROJECT_VERSION_MAJOR} MATCHES UNKNOWN) 
-    SET(PROJECT_VERSION_MAJOR_CONFIG "${PROJECT_NAME}_VERSION_UNKNOWN_TAG")
+  IF(NOT ${PROJECT_VERSION_MAJOR} MATCHES UNKNOWN) 
```

Comment:
I'm sorry, why the `NOT`? I think it's a leftover, it should be removed, isn't it?